toolset/setup/linux/languages/php.sh

```diff
@@ -49,6 +49,8 @@ printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f redis
 # yaf.so
 printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f yaf
```

Comment:
I think it's here because it requires `yaf.so` in the PHP config e.g. turning on PHP would result in errors if yaf wasn't installed. To pull it into a separate file you'd need to modify the php config from the yaf installer